tests/fixtures/integration/actions/config/test_direct_interactive_ee.py/test/6.json

```diff
@@ -5,33 +5,6 @@
-        "14│- This affects vars_files, include_vars, inventory and vars plugins among others.                                                                                                                                                                                                                       ▒",
-        "15│env:                                                                                                                                                                                                                                                                                                    ▒",
-        "16│- name: ANSIBLE_YAML_FILENAME_EXT                                                                                                                                                                                                                                                                       ▒",
-        "17│ini:                                                                                                                                                                                                                                                                                                    ▒",
-        "18│- key: yaml_valid_extensions                                                                                                                                                                                                                                                                            ▒",
-        "19│  section: defaults                                                                                                                                                                                                                                                                                     ▒",
-        "20│name: Yaml filename extensions                                                                                                                                                                                                                                                                          ▒",
-        "21│option: YAML_FILENAME_EXTENSIONS                                                                                                                                                                                                                                                                        ▒",
-        "22│source: default",
-        "^b/PgUp page up                          ^f/PgDn page down                          ↑↓ scroll                          esc back                          - previous                          + next                          [0-9] goto                          :help help"
-    ]
+        "compared_fixture": false
```

Comment:
There should be something in the present list if compare fixture is set to false.

Reply:
see above about the output key